src/Microsoft.ManagedZLib/tests/DeflateStreamTests.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.ManagedZLib.Tests;
+
+public class DeflateStreamTests
+{
+ 
+    public static IEnumerable<object[]> UncompressedTestFiles()
+    {
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.doc") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.docx") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.pdf") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "TestDocument.txt") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "alice29.txt") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "asyoulik.txt") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "cp.html") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "fields.c") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "grammar.lsp") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "kennedy.xls") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "lcet10.txt") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "plrabn12.txt") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "ptt5") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "sum") };
+        yield return new object[] { Path.Combine("UncompressedTestFiles", "xargs.1") };
```

Comment:
Do this for all the lines:
`yield return new object[] { "<only keep the filename>" };`

The test parameters would now be only one (the filename, remove the folder name).

Then adjust the rest of the code that gets you the file path so that it also takes the "UncompressedTestFiles" argument in its internal Path.Combine.

Reply:
OK the current fix is good too. The method that forms the path was part of the benchmarks as you mentioned.